Makefile

```diff
@@ -60,7 +60,7 @@ all: build
 build: $(BINDIR)/$(BINNAME)
 
 $(BINDIR)/$(BINNAME): $(SRC)
-	GO111MODULE=on go build $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' -o $(BINDIR)/$(BINNAME) ./cmd/helm
+	GO111MODULE=on go build $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' -o '$(BINDIR)'/$(BINNAME) ./cmd/helm
```

Comment:
I'm no expert but should...

```
'$(BINDIR)'/$(BINNAME)
```

be...

```
'$(BINDIR)/$(BINNAME)'
```

Notice the `'` is moved to the end of the path.

Reply:
It doesn't matter since the BINNAME doesn't contain spaces. The shell will treat any strings touching one another as a single string. For example, these all work:
`
# Quotes around everything:
cd "/usr/home/chris/Projects/Open Source/workspace"
# Quotes around base path:
cd "/usr/home/chris/Projects/Open Source"/workspace
# Quotes around only the directory containing the space:
cd /usr/home/chris/Projects/"Open Source"/workspace
# Quotefest 2020:
cd "/usr"/"home/"chris"/""Projects/""Open Source""/workspace"
`